client/src/components/UploadPhoto/UploadPhoto.tsx

```diff
@@ -0,0 +1,50 @@
+import Card from '@material-ui/core/Card';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+import DeleteForeverIcon from '@material-ui/icons/DeleteForever';
+
+import { Box, Button } from '@material-ui/core';
+import { useStyles } from './useStyles';
+
+export default function RecipeReviewCard() {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.root}>
+      <Box
+        className={classes.card}
```

Comment:
You can probably add these styles directly to the `Box` components through its props.